.github/workflows/test-webservice.yml

```diff
@@ -26,7 +26,7 @@ defaults:
 jobs:
   macos:
     name: macOS ${{ matrix.configuration }}
-    runs-on: macos-11
+    runs-on: macos-12
```

Comment:
Not sure, but I don't think there are any macOS-12 runner pools yet.

Reply:
See https://github.com/Apodini/Apodini/pull/317#discussion_r672958871